model-optimizer/mo/front/extractor.py

```diff
@@ -312,6 +314,8 @@ def update_ie_fields(attrs: dict, ir_version = None):
                         'coeff',
                         ('ratio', lambda node: attr_getter(node, 'ratio')),
                         'size',
+                        ('shape', lambda node: ','.join([str(i) for i in node.shape]) if node.has_valid('shape') else None),
+                        ('element_type', lambda node: np_data_type_to_destination_type(node.data_type) if node.has_valid('data_type') else None),
```

Comment:
We should not modify this list. It is a legacy approach which we do not rely on anymore. Instead modify the code which cuts the model and create Parameter nodes using Parameter class.

Reply:
Ok, done.